src/common/io/tls/client.h

```diff
@@ -0,0 +1,63 @@
+/***********************************************************************************************************************************
+TLS Client
+
+This is a simple, secure TLS client intended to allow access to services that are exposed via HTTPS.  We call is TLS instead of SSL
```

Comment:
Typo: "We call is TLS" should be "We call it TLS"

Reply:
Fixed.